src/app_service/service/contacts/service.nim

```diff
@@ -178,7 +189,7 @@ QtObject:
     result = self.fetchContact(id)
     if result.id.len == 0:
       let alias = self.generateAlias(id)
-      let identicon = self.generateIdenticon(id)
+      let identicon = self.imageServerUrl & "identicons?publicKey=" & id
```

Comment:
@richard-ramos yes, this line should be added to the `generateIdenticon` proc.

Reply:
I made the change in `generateIdenticon` of contacts service, but left `generateIdenticonFromPk` unmodified since the http server is not available before login in, and this function is invoked in the login/onboarding screens.